content/sifixes/src/npcs/myauri.uc

```diff
@@ -0,0 +1,36 @@
+/*  Copyright (C) 2016  The Exult Team
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ */
+ 
+/*  Myauri's MET flag never gets set during conversation with him, so this
+ *  bit of code will set it once you start talking to him. He does not seem
+ *  to check it either, so the only effect this code has is single-clicking
+ *  on him will show his name rather than the shape name "gwani".
+ *
+ *  2016-07-15 Written by Knight Captain
+ */
+
+void Myauri object#(0x492) () // NPC 146
+{
+	if (event == STARTED_TALKING) // DoubleClick would also work here.
+	{
+		if (!MYAURI->get_item_flag(MET)) // If we have not met Myauri,
+            {
+            MYAURI->set_item_flag(MET); // Now we have.
```

Comment:
i think this can be set unconditionally without first checking if MET is set or not.
so the code becomes:
```c
if (event == STARTED_TALKING)  MYAURI->set_item_flag(MET);
Myauri.original();
```